docs/source/release_notes/1.16.rst

```diff
@@ -1,54 +1,62 @@
+* The many-ports-open boefje/normalizer has been removed.
+* Backup scripts to backup container data when using the development setup. Thanks to @TobiasBDO for contributing the scripts.
 * More Octopoes Query support for complex path queries
+* Optimize queries executed when running bits
 * Introduce support for running custom built OCI images using only a boefje definition (boefje.json), applied to nmap.
-* Improvements of the design, plugin overview and Report titles.
-* Improvements of several Reports in terms of performance, styling, OOI selection and configuration.
-* More documentation on: Reports, the new OCI image functionality and architecture, IPv6 support in Docker and Octopoes Models.
+* Improvements of several reports in terms of performance, styling, OOI selection and configuration.
+* Improved documentation about reports, the new OCI image functionality and architecture, IPv6 support in Docker and Octopoes models.
+* Added documentation on how to make a boefje, normalizer, model, bit and report with examples. Thanks to @Souf149 for contributing some of these improvements.
+* Frysk has been added to the selectable list of languages.
 
 Bug fixes
 =========
 
-* Fix OOI Add/Edit form
-* Fix version handling when no version is present.
-* Fix aggregate plugin overview table
-* Fix task api status code response for malformed id in the scheduler
-* Fix select all OOIs
-* Fix openssl boefje being stuck on port 80
-* Fix pdf alignment
-* Fix critical vulnerability counter
-* Fix in System Specific Reports
-* fix schema errors on empty / missing schemas
-* Fix improve error handling
-* Fix missing cipher csv in Debian package
-* Fix Update nuclei
-* Fix and improve running boefjes/normalizer
-* Fix the KATalogus plugin API limit
+* Fix OOI add/edit form
+* Fix version handling when no version is present in wappalyzer normalizer.
+* Error handling has been improved in a lot of places.
+* Fixed schema errors when plugin schema is empty or missing.
+* Fix and improve manually running a boefjes/normalizer.
+* The KATalogus plugin API doesn't have a hardcoded limit anymore.
+* Missing titles in FastAPI API's have been added.
+* Added workaround for broken links to OOIs in the normalizer task list.
+* Fix Snyk boefje creating empty CVE ids.
 
 Upgrading
 =========
 
-It is no longer needed to seed the KATalogus database using `python -m boefjes.seed` on upgrades.
-This is because v1.16.0 phases out the `repository` database model in the KATalogus.
-The migration could potentially not be backward compatible for each install,
-So please read the following carefully before triggering an upgrade.
+It is no longer needed to seed the KATalogus database using `python -m
+boefjes.seed` on installation or upgrades. This is because v1.16.0 phases out
+the `repository` database model in the KATalogus. The migration could
+potentially not be backward compatible for each install, so please read the
+following carefully before triggering an upgrade.
 
```

Comment:
I think There's also a  need to rerun al bits because of some changes in those models. @noamblitz is that correct and already merged?

Reply:
That's correct!

Reply:
I added a sentence that you need to rerun all bits.